server/data/restClient.ts

```diff
@@ -10,7 +10,7 @@ import { restClientMetricsMiddleware } from './restClientMetricsMiddleware'
 
 interface GetRequest {
   path?: string
-  query?: string
+  query?: Record<string, string>
```

Comment:
The query string params need to be a map of name value pairs; rather than a string